src/Microsoft.ML.Core/Environment/HostEnvironmentBase.cs

```diff
@@ -392,6 +392,9 @@ public void RemoveListener(Action<IMessageSource, TMessage> listenerFunc)
 
             // This fork shares some stuff with the master.
             Master = source;
```

Comment:
Should we rename "Master" to something else?

Reply:
Yep I think so

Reply:
Its a protected member of an internal class. We should be able to rename it just fine without any breaking changes so it shouldn't have to wait for a major release. That being said, it would be better to do that in a separate PR since its not really related to any of these ObjDet changes.

Reply:
@JakeRadMSFT could you make an issue so we can track it please?

Reply:
https://github.com/dotnet/machinelearning/issues/6622